projects/examples/src/components/datagrid/datagrid-header.example.module.ts

```diff
@@ -0,0 +1,19 @@
+/*!
```

Comment:
Let's please create separate HTML, CSS and TS files so we can get syntax highlighting in our examples app